client/state/timezones/reducer.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import { TIMEZONES_RECEIVE } from 'state/action-types';
+
+import {
+	rawOffsetsSchema,
+	labelsSchema,
+	continentsSchema
+} from './schema';
+
+export const rawOffsetReducer = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, { rawOffsets = [] } ) => rawOffsets
```

Comment:
set-and-replace by the moment. The main purpose of this whole implementation is to be consistent with the wp-admin. For this reason, we did the `./timezones` endpoints, etc. Let's avoid the chance of not to be, at least for now.